roles/common/tasks/main.yml

```diff
@@ -23,3 +23,7 @@
 - name: assemble the authorized keys file
   assemble: dest=/root/.ssh/authorized_keys mode=0600 src=/tmp/pubkeys
   sudo: yes
+
+- name: install rackspace monitoring agent
+  script: monitoring.sh {{ rackspace_username }} {{ rackspace_api_key }}
```

Comment:
Yeah, I'm sure. I'll open an issue about it so I remember to fix it at some point, but I'm going to leave it as is at least for now.